app/data_transfer/dayone/models.py

```
@@ -0,0 +1,216 @@
+    locality_name: Optional[str] = Field(None, alias="localityName", max_length=200)
+    administrative_area: Optional[str] = Field(None, alias="administrativeArea", max_length=200)
+    country: Optional[str] = Field(None, alias="country", max_length=100)
+    time_zone_name: Optional[str] = Field(None, alias="timeZoneName", max_length=100)
+    region: Optional[Dict[str, Any]] = Field(None, alias="region")
+
+    @field_validator("latitude", "longitude")
+    @classmethod
+    def validate_coordinates(cls, v: Optional[float]) -> Optional[float]:
+        """Validate coordinates are finite numbers."""
+        if v is not None and not (-180 <= v <= 180):
+            raise ValueError("Coordinates must be within valid range")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)